client/my-sites/site-settings/form-general.jsx

```diff
@@ -889,7 +889,7 @@ const connectComponent = connect( ( state ) => {
 		isPaidPlan: isCurrentPlanPaid( state, siteId ),
 		isUnlaunchedSite: isUnlaunchedSite( state, siteId ),
 		isWPForTeamsSite: isSiteWPForTeams( state, siteId ),
-		isStagingSite: isSiteStaging( state, siteId ),
+		isWpcomStagingSite: isSiteWpcomStaging( state, siteId ),
```

Comment:
Would it make sense to use `isSiteWpcomStaging` as a variable name everywhere? Now we have `isWpcomStagingSite` and `isWpcomStaging`, but the selector name is `isSiteWpcomStaging`.

For other selectors, we usually use the variable name the same as the selector name.

Reply:
Thanks for the explanation. It makes sense just to unify `isWpcomStagingSite` and `isWpcomStaging` then.